dbt/include/redshift/macros/relations/table/replace.sql

```diff
@@ -0,0 +1,47 @@
+{% macro redshift__get_replace_table_sql(relation, sql) -%}
+
+  {%- set _dist = config.get('dist') -%}
+  {%- set _sort_type = config.get(
+          'sort_type',
+          validator=validation.any['compound', 'interleaved']) -%}
+  {%- set _sort = config.get(
+          'sort',
+          validator=validation.any[list, basestring]) -%}
+  {%- set sql_header = config.get('sql_header', none) -%}
+  {%- set backup = config.get('backup') -%}
+
+  {{ sql_header if sql_header is not none }}
+
+  {%- set contract_config = config.get('contract') -%}
+  {%- if contract_config.enforced -%}
+
+  create table
```

Comment:
These (this line and line 36) should be `create or replace` instead of just `create`.

Reply:
I don't think redshift supports `or replace`

Reply:
If Redshift doesn't support `create or replace` for tables, then we need to remove `RelationType.Table` from `replaceable_relations` and then delete this file.

Reply:
good point! duh